src/java/fr/paris/lutece/plugins/forms/export/csv/CSVUtil.java

```diff
@@ -83,6 +83,9 @@ public static String buildColumnName( Question question )
      */
     static String safeString( String strValue )
     {
-        return StringEscapeUtils.escapeCsv( strValue );
+    	if (strValue == null) {
+    		strValue = StringUtils.EMPTY;
+    	}
+    	return "\"" + strValue + "\"";
```

Comment:
so what if `strValue `contains a `"` character ?

Reply:
" character is forbidden in a response form. You can't write this char on a text area

Reply:
I tried with a response string containing a " char (editing the content on the DataBase) : no issues.